### PR TITLE
add update_loss_scaling npu min value

### PR DESCRIPTION
**Pull request by Baibaifan**
### PR types

New features
### PR changes

OPs
### Describe

Add npu LossScaling min value
We added an environment variable "FLAGS_min_loss_scaling" to set the minimum value of updated loss scaling tensor.
If you want to use, you can export FLAGS_min_loss_scaling=xxx before your run program.

image

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/npu/test_update_loss_scaling_min_op_npu.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        found_inf = np.array([True], dtype=np.bool)
+        x = np.random.random((1024, 1024)).astype(self.dtype)
+        i = np.random.randint(0, 1024, 1)
+        j = np.random.randint(0, 1024, 1)
+        x[i[0]][j[0]] = np.inf
+
+        self.inputs = {
+            'X': [('x0', x)],
+            'FoundInfinite': found_inf,
+            'PrevLossScaling': self.prev_loss_scaling,
```
Comment (gongweibao) [inline python/paddle/fluid/tests/unittests/npu/test_update_loss_scaling_min_op_npu.py:47]: 这些值要显式的赋值一下。

**Review by gongweibao — CHANGES_REQUESTED** [review]

File: python/paddle/fluid/tests/unittests/npu/test_update_loss_scaling_min_op_npu.py (hunk 1 above)
Comment (Baibaifan) [reply python/paddle/fluid/tests/unittests/npu/test_update_loss_scaling_min_op_npu.py:47]: Done

**Review by gongweibao — DISMISSED** [review]
LGTM
修改一下PR的标题，cherrypick到ascendrelease分支!

**Review by zhiqiu — APPROVED** [review]
LGTM

**Review by phlrain — APPROVED** [review]

**Pull request merged**